src/Game.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import {RockButton} from "./RockButton";
+import {PaperButton} from "./PaperButton";
+import {ScissorsButton} from "./ScissorsButton";
+import {WaitButton} from "./WaitButton";
+import {ResultBanner} from "./ResultBanner";
+
+let houseComponent;
+
+function Game({scoreNumber, handleScoreNumber}) {
+    const [gameStatus, setGameStatus] = React.useState('initial');
+    const [buttonPicked, setButtonPicked] = React.useState('');
+    const [housePicked, setHousePicked] = React.useState('');
```

Comment:
`housePicked` can be a boolean state instead of ''/'picked'

Reply:
done